exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -167,18 +220,23 @@ def collector(self):
         )
         return self._collector
 
-    def export(self, spans):
-        jaeger_spans = _translate_to_jaeger(spans)
-
-        batch = jaeger.Batch(
-            spans=jaeger_spans,
-            process=jaeger.Process(serviceName=self.service_name),
-        )
-
-        if self.collector is not None:
-            self.collector.submit(batch)
+    def export(self, spans) -> SpanExportResult:
+        if self.transport_format == TRANSPORT_FORMAT_PROTOBUF:
+            jaeger_spans = protobuf._to_jaeger(spans, self.service_name)
+            batch = model_pb2.Batch(spans=jaeger_spans)
+            request = PostSpansRequest(batch=batch)
```

Comment:
Maybe not, but can these 3 lines be moved into the next `if`?

Reply:
The next `if` is to make sure client exists. If collector endpoint is not set for `thrift` It uses agent with with default values. Should proto also take default endpoint if not provided, If so we can get rid of next `if`.

Reply:
I think this if is justified, IIUC the point is that values from lines 225-227 aren't used if the second if isn't true. I see @ocelotl's point as well: it seems the above three can be moved to the below if.

We can't get rid of this if (I don't believe we can accept default protobuf values), but we can reduce unneeded work if it's not true.

Reply:
Agent has default host `localhost` and port as `6381` when exporter is instantiated and if nothing is supplied it uses the default values. If we can do the same for proto defaults which is `localhost:14250` (Jaeger default for proto) we can get rid of this. Thoughts?

Reply:
The spec also has default to `localhost:14250`. I think we can use the default host:port for grpc if endpoint param is not set.